### ISSUE TITLE
Add Docker Hub Pulls and Github Stars for each of the tools

### ISSUE THREAD

**Issue by collabnix** [collabnix/dockertools#39]
This is a feature request to add Docker Hub pulls as well as GitHub Stars for each of these tools.

I have already added for the first section. Have a look.

**Comment by collabnix** [comment]
@0904-mansi

**Comment by collabnix** [comment]
Any update?

**Comment by 0904-mansi** [comment]
> Any update?

Sir, I added in most of the tools but suddenly many of the docker pull badges are showing invalid repo don't know why because they were showing properly when I added.

**Comment by collabnix** [comment]
Hi @0904-mansi I have removed Docker Hub Pulls stats from each of these tools as it seems to be unstable.

**Comment by 0904-mansi** [comment]
> Hi @0904-mansi I have removed Docker Hub Pulls stats from each of these tools as it seems to be unstable.

Thank you, sir Now I should add only stars for each tool right?

**Comment by collabnix** [comment]
Yes. Also, it would be great if you can find the way or tools that helps us track how many users are clicking on these tools or which are the popular tools?

**Comment by 0904-mansi** [comment]
> Yes. Also, it would be great if you can find the way or tools that helps us track how many users are clicking on these tools or which are the popular tools?

ok sir I'll try.
